sql/moz-fx-data-shared-prod/fenix_derived/funnel_retention_clients_week_2_v1/metadata.yaml

```diff
@@ -8,9 +10,10 @@ labels:
   schedule: daily
   incremental: true
   table_type: client_level
-scheduling:
-  dag_name: bqetl_analytics_tables
-  depends_on_past: false
+# @kik 2025-02-18: descheduled due to deprecation.
```

Comment:
We could consider unscheduling as one of the follow up steps if we think this data may still be in use.